### PR TITLE
Mezzo GA

### PR DESCRIPTION
**Pull request by davidmu1**

### CODE REVIEW

File: api-reference/v1.0/api/orgcontact-list-transitivememberof.md [hunk 1]
```diff
@@ -56,33 +56,14 @@ If successful, this method returns a `200 OK` response code and a collection of
...
 
-# [HTTP](#tab/http)
 <!-- {
   "blockType": "request",
   "name": "orgcontact_list_transitivememberof"
 }-->
 
 ```msgraph-interactive
-GET https://graph.microsoft.com/v1.0/me/transitiveMemberOf
+GET https://graph.microsoft.com/v1.0/{id}/transitiveMemberOf
```
Comment (millicentachieng) [inline api-reference/v1.0/api/orgcontact-list-transitivememberof.md:65]: Should match what's under the **HTTP request** heading  `https://graph.microsoft.com/v1.0/contacts/{id}/transitiveMemberOf`

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 2]
```diff
@@ -37,13 +37,14 @@ GET /groups/{id}/transitiveMembers
 
 ## Optional query parameters
 
-This method supports the [OData query parameters](/graph/query-parameters) to help customize the response.
+This method supports the [OData query parameters](/graph/query-parameters) to help customize the response, including `$search`, `$count`, and `$filter`. You can use `$search` on the **displayName** property. When items are added or updated for this resource, they are specially indexed for use with the `$count` and `$search` query parameters. There can be a slight delay between when an item is added or updated and when it is available in the index.
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:40]:
```suggestion
This method supports the [OData query parameters](/graph/query-parameters) to help customize the response, including `$search`, `$count`, and `$filter`. You can use `$search` on the **displayName** and **description** properties. When items are added or updated for this resource, they are specially indexed for use with the `$count` and `$search` query parameters. There can be a slight delay between when an item is added or updated and when it is available in the index.
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 3]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
-      "givenName": "givenName-value",
-      "jobTitle": "jobTitle-value",
-      "mail": "mail-value",
-      "mobilePhone": "mobilePhone-value",
-      "officeLocation": "officeLocation-value",
-      "preferredLanguage": "preferredLanguage-value",
-      "surname": "surname-value",
-      "userPrincipalName": "userPrincipalName-value",
-      "id": "id-value"
+      "id": "11111111-2222-3333-4444-555555555555",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:112]:
```suggestion
      "@odata.type": "#microsoft.graph.user",
      "id": "11111111-2222-3333-4444-555555555555",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 4]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+#### Response
+
+The following is an example of the response.
+
+>**Note:** The response object shown here might be shortened for readability. All the properties will be returned from an actual call.
+
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "microsoft.graph.directoryObject",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:147]: raw count not a directory object

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 5]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+  "isCollection": true
+} -->
+```http
+HTTP/1.1 200 OK
+Content-type: text/plain
+```
+
+893
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:155]:
```suggestion
`893`
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 6]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+#### Response
+
+The following is an example of the response.
+
+>**Note:** The response object shown here might be shortened for readability. All the properties will be returned from an actual call.
+
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "microsoft.graph.group",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:182]:
```suggestion
  "@odata.type": "microsoft.graph.user",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 7]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+  "isCollection": true
+} -->
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#groups(displayName,id)",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:190]:
```suggestion
  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#users(displayName,id)",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 8]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
-      "jobTitle": "jobTitle-value",
-      "mail": "mail-value",
-      "mobilePhone": "mobilePhone-value",
-      "officeLocation": "officeLocation-value",
-      "preferredLanguage": "preferredLanguage-value",
-      "surname": "surname-value",
-      "userPrincipalName": "userPrincipalName-value",
-      "id": "id-value"
+      "id": "11111111-2222-3333-4444-555555555555",
+      "mail": "group1@contoso.com",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:113]:
```suggestion
     "displayName": "Joseph Price",
     "mail": "Joseph.Price@contoso.com",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 9]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+      "mailEnabled": true,
+      "mailNickname": "Contoso1",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:115]: remove mailEnabled, mailNickname and securityEnabled: not relevant for User Object

File: api-reference/v1.0/api/group-list-members.md [hunk 10]
```diff
@@ -87,12 +96,172 @@ The following is an example of the response.
...
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#groups",
+  "@odata.count":76,
+  "value":[
+    {
+      "displayName":"AAD Contoso Users",
+      "mail":"AADContoso_Users@contoso.com",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-members.md:261]:
```suggestion
      "mail":"Joseph.Price@contoso.com",
```

File: api-reference/v1.0/api/group-list-members.md [hunk 11]
```diff
@@ -87,12 +96,172 @@ The following is an example of the response.
...
+      "mailEnabled":true,
+      "mailNickname":"AADContoso_Users",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-members.md:263]: remove mailEnabled, mailNickname and securityEnabled: not relevant for User Object

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 12]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#groups(displayName,id)",
+  "@odata.count":7,
+  "value":[
+    {
+      "displayName":"Contoso-tier Query Notification",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:194]:
```suggestion
      "displayName":"Joseph Price",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 13]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+#### Response
+
+The following is an example of the response.
+
+>**Note:** The response object shown here might be shortened for readability. All the properties will be returned from an actual call.
+
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "microsoft.graph.group",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:226]:
```suggestion
  "@odata.type": "microsoft.graph.user",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 14]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+  "isCollection": true
+} -->
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#groups",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:234]:
```suggestion
  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#users",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 15]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+  "@odata.count":76,
+  "value":[
+    {
+      "displayName":"AAD Contoso Users",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:238]:
```suggestion
      "displayName":"Joseph Price",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 16]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+      "mail":"AADContoso_Users@contoso.com",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:239]:
```suggestion
      "mail":"Joseph.Price@contoso.com",
```

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 17]
```diff
@@ -105,25 +109,141 @@ Content-type: application/json
...
+      "mailEnabled":true,
+      "mailNickname":"AADContoso_Users",
```
Comment (Licantrop0) [inline api-reference/v1.0/api/group-list-transitivemembers.md:241]: remove mailEnabled, mailNickname and securityEnabled: not relevant for User Object

File: api-reference/v1.0/api/serviceprincipal-list-memberof.md [hunk 18]
```diff
@@ -34,14 +34,16 @@ GET /servicePrincipals/{id}/memberOf
 
 ## Optional query parameters
 
-This method supports the [OData Query Parameters](https://developer.microsoft.com/graph/docs/concepts/query_parameters) to help customize the response.
+This method supports the [OData query parameters](/graph/query_parameters) to help customize the response, including `$search`, `$count`, and `$filter`. OData cast is also enabled, for example, you can cast to get just the directoryRoles the user is a member of. You can use `$search` on the **displayName** property. When items are added or updated for this resource, they are specially indexed for use with the `$count` and `$search` query parameters. There can be a slight delay between when an item is added or updated and when it is available in the index.
```
Comment (Licantrop0) [inline api-reference/v1.0/api/serviceprincipal-list-memberof.md:37]:
```suggestion
This method supports the [OData query parameters](/graph/query_parameters) to help customize the response, including `$search`, `$count`, and `$filter`. OData cast is also enabled, for example, you can cast to get just the directoryRoles the user is a member of. You can use `$search` on the **displayName** and **description** properties. When items are added or updated for this resource, they are specially indexed for use with the `$count` and `$search` query parameters. There can be a slight delay between when an item is added or updated and when it is available in the index.
```

**Review by Licantrop0 — CHANGES_REQUESTED** [review]
Addressing issues in queries and other details.

**Review by Licantrop0 — APPROVED** [review]
few issues with metadata, but we can send it and fix later

**Review by kksrivatsa — APPROVED** [review]

File: api-reference/v1.0/api/device-list-memberof.md [hunk 19]
```diff
@@ -106,9 +114,129 @@ Content-type: application/json
...
+
+### Example 4: Use OData cast and $filter to get membership with a display name that starts with the letter 'A' including a count of returned objects
+
+#### Request
+
+The following is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "get_a_count"
```
[the remark this answers is not in the document]
Comment (davidmu1) [reply api-reference/v1.0/api/device-list-memberof.md:198]: Not seeing a duplicate name in the file.

File: concepts/changelog.md [hunk 20]
```diff
@@ -96,6 +96,12 @@ For a summary of the value of these API changes, as well as recent tools, compon
...
...st servicePrincipal memberOf](/graph/api/serviceprincipal-list-memberof?view=graph-rest-1.0)</li><li>[List servicePrincipal transitive memberOf](/graph/api/serviceprincipal-list-transitivememberof?view=graph-rest-1.0)</li><li>[List users](/graph/api/user-list?view=graph-rest-1.0)</li><li>[List user memberOf](/graph/api/user-list-memberof?view=graph-rest-1.0)</li><li>[List user transitive memberOf](/graph/api/user-list-transitivememberof?view=graph-rest-1.0</li></ul><br>Added examples for using OData cast to: <ul><li>[List group memberOf](/graph/api/group-list-memberof?view=graph-rest-1.0)</li><li>[List group members](/graph/api/group-list-members?view=graph-rest-1.0)</li><li>[List group transitive memberOf](/graph/api/group-list-transitivememberof?view=graph-rest-1.0)</li><li>[List servicePrincipal memberOf](/graph/api/serviceprincipal-list-memberof?view=graph-rest-1.0)</li><li>[List servicePrincipal transitive memberOf](/graph/api/serviceprincipal-list-transitivememberof?view=graph-rest-1.0)</li><li>[List user memberOf](/graph/api/user-list-memberof?view=graph-rest-1.0)</li><li>[List user transitive memberOf](/graph/api/user-list-transitivememberof?view=graph-rest-1.0)</li></ul> |
```
Comment (Lauragra) [inline concepts/changelog.md:103]: @davidmu1 , I assume the version here should be v1.0, since the update is to v1.0 files?

**Pull request merged**